src/item.cpp

```diff
@@ -2237,12 +2264,16 @@ bool item::is_auxiliary_gunmod() const
 
 int item::get_storage() const
 {
-    const auto t = find_armor_data();
+    float pockets = 1;
+    auto t = find_armor_data();
     if( t == nullptr ) {
         return 0;
     }
+    if (pocketed == true){
+        pockets = (1.5); //Test value
+    }
     // it_armor::storage is unsigned char
-    return static_cast<int>( static_cast<unsigned int>( t->storage ) );
+    return (static_cast<int> (static_cast<unsigned int>( t->storage * pockets) ) );
```

Comment:
> Why did you remove the const from t, you don't change it, so why does it have to be non-const?

I removed it when my initial implementation was to change t. I'll add it back. And I'll clean up the brackets